internal/backup_util.go

```diff
@@ -113,6 +140,27 @@ func SortBackupTimeSlices(backupTimes []BackupTime) {
 	})
 }
 
+func SortBackupTimeWithMetadataSlices(backupTimes []BackupTimeWithMetadata) {
+	order := ByCreationTime
```

Comment:
Well, if we always sort by the creation time why separate if down below? I suggest to remove it